src/app/ui-library/callout.tsx

```diff
@@ -0,0 +1,94 @@
+import { CheckCircle, Info, Warning, WarningCircle } from '@phosphor-icons/react';
+import { createElement } from 'react';
+import styled from 'styled-components';
+import Theme from 'theme';
+import { StyledHeading, StyledP } from './common.styled';
+
+type CalloutVariant = 'info' | 'warning' | 'danger' | 'success';
+
+const backgroundColors = {
+  info: Theme.colors.elevation6_600,
+  warning: Theme.colors.caution_800,
+  success: Theme.colors.success_medium_700,
+  danger: Theme.colors.danger_dark_600,
+};
+const getBackgroundForVariant = (variant: CalloutVariant) => backgroundColors[variant];
+
+const icons = {
+  info: Info,
+  warning: WarningCircle,
+  success: CheckCircle,
+  danger: Warning,
+};
+const getIconForVariant = (variant: CalloutVariant) =>
+  createElement(icons[variant], {
+    weight: 'fill',
+    size: '24',
+    color: Theme.colors.white_200,
+  });
+
+const Container = styled.div<{ variant: CalloutVariant }>`
+  display: flex;
+  flex-direction: row;
+  border-radius: ${(props) => props.theme.radius(2)}px;
+  align-items: flex-start;
+  background-color: ${(props) => getBackgroundForVariant(props.variant)};
+  padding: ${(props) => props.theme.spacing(8)}px;
+  gap: ${(props) => props.theme.spacing(8)}px;
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const RedirectButton = styled.button`
+  background-color: transparent;
+  color: ${(props) => props.theme.colors.white_0},
+  display: flex;
+  margin-top: 4;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * ref: https://zeroheight.com/0683c9fa7/p/051ca8-callout/t/7814dc
+ */
+export type CalloutProps = {
+  titleText?: string;
+  bodyText: string;
+  variant?: CalloutVariant;
+  redirectText?: string;
+  onClickRedirect?: () => void;
+};
+export function Callout({
+  titleText,
+  bodyText,
+  variant = 'info',
+  redirectText,
+  onClickRedirect = () => {},
+}: CalloutProps) {
+  const icon = getIconForVariant(variant);
+  return (
+    <Container variant={variant}>
```

Comment:
Aren't you getting errors in the console for these? Something about 'are you sure this is supposed to be a prop'? I think styled components need a $ infront of the var name
https://styled-components.com/docs/basics#passed-props

Reply:
I don't see these console errors.. do you?

maybe it's only necessary to include $ if the prop name matches a valid attribute name?

Reply:
Interesting. Maybe it was only introduced in version 6 🤔 Will mark as resolved 👍